demux/demux_lavf.c

```diff
@@ -423,6 +423,19 @@ static char *remove_prefix(char *s, const char *const *prefixes)
 static const char *const prefixes[] =
     {"ffmpeg://", "lavf://", "avdevice://", "av://", NULL};
 
+/*
+ * https://web.archive.org/web/20230512195331/
+ *     https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/StreamingMediaGuide/
+ *     DeployingHTTPLiveStreaming/DeployingHTTPLiveStreaming.html
+ *
+ * https://www.rfc-editor.org/rfc/rfc8216.txt
+ */
```

Comment:
that long URL is kind of unhandy and IMO we don't need to cite a source for some mimetypes, so feel free to remove